### PR TITLE
Add simple Dockerization

### PR DESCRIPTION
**Pull request by AgentScrubbles**
Adding simple dockerfile and sample docker compose for integrating automod with the standard Lemmy stack.

If `threativore` is cloned to the same directory as where lemmy is running, the docker compose is extremely easy

```
  automod:
    build:
      context: ./threativore
      dockerfile: Dockerfile
    networks:
      - lemmyinternal
    environment:
      - LEMMY_DOMAIN=<< instance URL>>
      - LEMMY_USERNAME=<<lemmy automod username>>
      - LEMMY_PASSWORD=<<lemmy automod password>>
      - USE_SQLITE=0
      - THREATIVORE_ADMIN_URL=<< instance admin user account >>
      - POSTGRES_USER=<< New: Postgres User Account >>
      - POSTGRES_URL=<< Host of existing postgres db >>
      - POSTGRES_PASS=<< PW of existing postgres db >>
    restart: always
```

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      - LEMMY_DOMAIN=
+      - LEMMY_USERNAME=
+      - LEMMY_PASSWORD=
+      - USE_SQLITE=1
+      - THREATIVORE_ADMIN_URL=
+    restart: always
+    ports:
+      - 8000:8080
+    volumes:
+      - ./volumes/automoderator:/usr/src/app/database
```
Comment (db0) [inline docker-compose.yml]: The default sqlite location is not this path. Where did you get this from?

**Review by db0 — CHANGES_REQUESTED** [review]
Can you potentially adjust the README for the docker part, or add a new docker readme? Do mention that the postgresql DB and user need to already exist

File: docker-compose.yml (hunk 1 above)
Comment (AgentScrubbles) [reply docker-compose.yml]: Oh crap, this was leftover from a copy of something else.  Where does it save it by default? I'll update it to that

Comment (db0) [reply docker-compose.yml]: https://github.com/db0/threativore/blob/38fbe78a7f15c76a34a03b16b0b16ad3dbec3739/threativore/flask.py#L17C46-L17C52

Typically it should be in `./instance/threativore.db`

Comment (AgentScrubbles) [reply docker-compose.yml]: I took a stab at it, but I could use a second pair of eyes on it.  I can verify 100% the postgres version works, but I don't have a test environment for the sqlite version.  Not to shlep it off, but that would be a great test for me, to see if you can spin up the container with sqlite and tell me it works (and that it persists after running it once).  It should literally be the same as the python version running it, you should be able to just point your volume to the directory your .db is in.

**Comment by AgentScrubbles** [comment]
Updated the documentation, but it is a bit wordy now.  If you would like I can split it out into a separate readme, but take a look at the version now
